datamining/views.py

```diff
@@ -66,8 +66,8 @@ def show_members_by_group(request):
     today = datetime.date.today().strftime("%Y-%m-%d")
 
     for member in found_members:
-        if len(member.email) > 0:
-            member_mails.append(member.email)
+        if len(member.get_email()) > 0:
```

Comment:
Doesn't member.get_email just return a singular email address?
I'd expect something akin to:
```
for member in found_members:
  email = member.get_email()
  if email is not None:
    member_mails.append(email)
```

Reply:
I'm not sure but that is what you wrote 2 months ago with the update of the member filter. I would have to check for None and Blank I would guess.

Reply:
oh lol, it's python so it checks for the length of the string xD

Reply:
Perhaps a simple `if email:` is correct here.

Reply:
I did together with none